test/it/IntegrationSpec.scala

```diff
@@ -67,4 +72,10 @@ abstract class IntegrationSpec
 
   protected def givenCacheFor(pid: String, answers: Answers): Unit = await(cacheRepository.insert(Cache.format.writes(Cache(pid, answers))))
 
+  protected def verifyEventually(requestPatternBuilder: RequestPatternBuilder): Unit = {
+    eventually {
+      Try(WireMock.verify(requestPatternBuilder)).isSuccess mustBe true
+    }
+  }
```

Comment:
Brackets for this function are unnecessary, but it's just style comment

Reply:
fixed